examples/jsm/tsl/display/MeshBlendNode.js

```diff
@@ -0,0 +1,124 @@
+import { TempNode, MeshBasicNodeMaterial, RenderTarget, RGBAFormat, NearestFilter, QuadMesh, Vector2 } from 'three/webgpu';
+import { nodeObject, vec4, vec3, float, modelPosition, modelWorldMatrix, passTexture, hash, Fn, wgslFn, NodeUpdateType, texture, screenUV } from 'three/tsl';
+
+//Source: https://www.jacktollenaar.top/mesh-seam-smoothing-blending#h.50wag6hqg9gh
+
+const _size = /*@__PURE__*/ new Vector2();
+
+class MeshBlendNode extends TempNode {
+    constructor(sceneOutputNode, sceneDepthNode, camera, scene) {
+        super('vec4');
+        this.sceneOutputNode = sceneOutputNode;
+        this.sceneDepthNode = sceneDepthNode;
+        this.updateBeforeType = NodeUpdateType.FRAME;
+        this.renderTarget = new RenderTarget(1, 1);
+        this.mainCamera = camera;
+        this.mainScene = scene;
+        this.blendFactor = float(1.2);
+        this.kernelSize = float(5);
+        this.kernelRadius = float(0.01 * this.blendFactor.value);   
+        this.depthFalloff = float(0.0001 * this.blendFactor.value);
+        this.debugMaterial = new MeshBasicNodeMaterial();
+        this._quadMesh = new QuadMesh(this.debugMaterial);
+    }
+
+    setup(){
+         const CustomHash = wgslFn(`
+            fn Hash(p: vec3f) -> f32 {
```

Comment:
Please implement all shader code as TSL. Otherwise it won't work with the WebGL backend.